node.js

```diff
@@ -18,11 +18,16 @@ module.exports = {
     es6: true,
```

Comment:
I'm finding naming `node.js` to be little confusing. Maybe I would rename it to `base.js`